src/widgets/SequentialSteps.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Ionicons from 'react-native-vector-icons/Ionicons';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+
+import { WARM_GREY, SUSSOL_ORANGE } from '../globalStyles';
+
+export default class SequentialSteps extends React.PureComponent {
+  renderStepIcon = ({ isCurrentStep, isLessThanCurrentStep, step }) => {
+    const { error } = step;
+    return (
+      <View style={{ minWidth: 40 }}>
+        {isCurrentStep && error && <Ionicons name="ios-close" color="red" size={30} />}
```

Comment:
what the use case for an error ?

Reply:
currently using it for an empty result - eg no periods are defined for this combination of program and order type

Reply:
added a doc string 4d7664bb0599396d3acd2563272644fd459c6ff7